.github/ISSUE_TEMPLATE/question.md

```diff
@@ -8,8 +8,6 @@ labels: "question"
 
 _Please note that this issue tracker is not a help form and this issue will be closed._
 
-Please check here instead:
+This is better suited to a [discussion](https://github.com/KotlinIsland/basedmypy/discussions).
+
```

Comment:
Can prob do it like this instead https://github.com/microsoft/TypeScript/blob/main/.github/ISSUE_TEMPLATE/config.yml#L6